iowrappers/users.go

```diff
@@ -85,7 +88,7 @@ func (redisClient *RedisClient) FindUser(context context.Context, findUserBy Fin
 	return userView, nil
 }
 
-func (redisClient *RedisClient) CreateUser(context context.Context, userView user.View) (user.View, error) {
+func (redisClient *RedisClient) CreateUser(context context.Context, userView user.View, skipPasswordGeneration bool) (user.View, error) {
```

Comment:
curious why we need `skipPasswordGeneration` with user email verification? Is it trying to skip password encryption? IIUC, the password is from user input, and we encrypted it before storing it on Redis. Does email verification process add some additional steps?

Reply:
> LGTM left one comment, is the PR deployed somewhere, so we can test it?

Deployed on https://staging-vp.herokuapp.com/v1/

Reply:
> 

Yes, the purpose is to skip password encryption. When a user signs up, we cannot store the original password but the encrypted one temporarily. So the user creation function has to deal with both cases.